Mathlib/Topology/Constructions.lean

```diff
@@ -1705,3 +1705,28 @@ theorem IsClosed.trans (hγ : IsClosed γ) (hβ : IsClosed β) : IsClosed (γ :
   convert IsClosed.inter hδ hβ
 
 end Monad
+
+section NhdsSet
+variable {X Y : Type*} [TopologicalSpace X] [TopologicalSpace Y] {f : Filter X}
+  {s : Set X} {t : Set Y} {x : X}
+
+theorem Filter.nhdsSet_prod_le_prod : 𝓝ˢ (s ×ˢ t) ≤ 𝓝ˢ s ×ˢ 𝓝ˢ t := by
```

Comment:
We have this lemma as [nhdsSet_prod_le](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Compactness/Compact.html#nhdsSet_prod_le)
Please move it here instead of adding it under a new name.